bin/make/e2e.sh

```diff
@@ -5,4 +5,29 @@ set -o errexit
 PLAYWRIGHT_VERSION=1.40.1
 UBUNTU_VERSION=jammy
 
-docker run --rm --network host -v $(pwd):/work/ -w /work/ -it mcr.microsoft.com/playwright:v$PLAYWRIGHT_VERSION-$UBUNTU_VERSION yarn test:e2e
+E2E_FLAG=""
+XVFB=""
+
+while [[ $# -gt 0 ]]; do
+    case "$1" in
+        --update)
+            E2E_FLAG=":update"
+            ;;
+        --ui)
+            E2E_FLAG=":ui"
+            # https://playwright.dev/docs/next/ci#running-headed
+            XVFB="xvfb-run"
+            ;;
+        --report)
+            E2E_FLAG=":report"
+            ;;
+        *)
+            # Unknown option
+            echo "Unknown option: $1"
+            exit 1
+            ;;
+    esac
+    shift
+done
+
+docker run --rm --network host -v $(pwd):/work/ -w /work/ -it mcr.microsoft.com/playwright:v$PLAYWRIGHT_VERSION-$UBUNTU_VERSION $XVFB yarn test:e2e$E2E_FLAG
```

Comment:
My implementation was shorter
```
#!/bin/sh

set -o errexit

PLAYWRIGHT_VERSION=1.39.0
UBUNTU_VERSION=jammy

docker run --rm --network host -v $(pwd):/work/ -w /work/ -it mcr.microsoft.com/playwright:v$PLAYWRIGHT_VERSION-$UBUNTU_VERSION yarn test:e2e$1
```

But I see you add more stuff.

Reply:
Yeah, you are right. But I have to fix the running UI in Docker, so that is why `XVFB` flag and so on. 🤷